libs/sdk-core/src/breez_services.rs

```diff
@@ -2031,10 +2032,20 @@ impl BreezServicesBuilder {
         );
 
         // breez_server provides both FiatAPI & LspAPI implementations
-        let breez_server = Arc::new(BreezServer::new(
-            self.config.breezserver.clone(),
-            self.config.api_key.clone(),
-        ));
+        let breez_server = Arc::new(
+            BreezServer::new(self.config.breezserver.clone(), self.config.api_key.clone())
+                .map_err(|e| ConnectError::Generic {
```

Comment:
Would it make sense to change this to a ServiceConnectivity error? I know it's technically not "connecting" at this point, but would be a misconfiguration on the breez server url.
```suggestion
                .map_err(|e| ConnectError::ServiceConnectivity {
```

Reply:
Makes sense. Done